builder/build.sh

```diff
@@ -64,6 +64,17 @@ fetch_r_source() {
   rm /tmp/R-${1}.tar.gz
 }
 
+install_libcurl() {
```

Comment:
A comment here (or in `set_up_environment`) that explains why we do this would be nice for future reference, even if we'll end up removing it in a year anyway.

Reply:
Or alternatively, download and compile this in the CentOS 7 image (but don't install it unless R >= 4.3)? Precompiling might shave off some build time as well.

Reply:
We could do that, but downloading and compiling libcurl takes about less than two  minutes, so I am not sure if it is worth it.

Reply:
In the end I did change it to build from the centos-7 Dockerfile, and only install in the build script on CentOS7 and only for R-devel.

The other thing that I'll still try is installing a newer libcurl version. AFAIK RedHat/CentOS backports bug fixes, so a stock 7.32.0 might be worse than a CentOS 7 7.29.x security-wise.

Reply:
OK, the latest libcurl version seems to work.